pkg/controllers/clusterresourceplacement/placement_status.go

```diff
@@ -30,6 +30,9 @@ var (
 
 const (
 	// ClusterResourcePlacementStatus condition reasons
+	validResourceSelectorsReason   = "ResourceEligible"
+	invalidResourceSelectorsReason = "ResourceIneligible"
```

Comment:
This is used to for checking if the resource selectors are good; personally I feel that ineligible might not be most accurate, e.g. failing due to invalid GVRs rather than selecting non-cluster-scoped resources.

Reply:
A simple `invalidResourceSelector` woud do I guess?